### PR TITLE
manual: db.in.ogr add column type detection for CSV

### LINKED ISSUE #4674: [Feat] Update db.in.ogr manual to improve import CSV files
### The problem
CSV is one of the most commonly and widely used formats for exchanging tabular data across platforms. However, it is difficult to import csv files into GRASS. Currently, db.in.ogr imports csv files but transforms all columns to text. In the current manual text, the suggested work around is to create an accompanying *.csvt file that specifies the data types for each column. This is very cumbersome, especially for files with many columns. But OGR will automatically recognize column data types if a file open option is set
`-oo AUTODETECT_TYPE=YES`
This makes importing csv files much easier. This can be manually entered into the **_gdal_oo_** argument in db.in.ogr but this is not mentioned in the current manual. 

### Proposed solution
While -oo AUTODETECT_TYPE=YES should be the default for importing a csv file using db.in.ogr (see #4593), until that change can be made, it would be very helpful to describe in the manual how this argument can be implemented as a workaround for now. The information in the current manual about using a *.csvt file should also be maintained for finer manual control. I propose the manual should be updated as follows: 

**Current:**
Import CSV file

Limited type recognition can be done for Integer, Real, String, Date, Time and DateTime columns through a descriptive file with same name as the CSV file, but .csvt extension (see details here).

NOTE: create koeppen_gridcode.csvt first for automated type recognition
```
db.in.ogr input=koeppen_gridcode.csv output=koeppen_gridcode
db.select table=koeppen_gridcode
```

**New:**
Import CSV file

db.in.ogr will attempt to automatically read input data types if the gdal_doo flag is set to 
AUTODETECT_TYPE=YES.
```
db.in.ogr input=koeppen_gridcode.csv output=koeppen_gridcode gdal_doo=AUTODETECT_TYPE=YES
db.select table=koeppen_gridcode
```
Users can also specify data types for CSV columns using a type definition file with same name as the CSV file, but *.csvt extension (see details here). Columns can be defined as Integer, Real, String, Date, Time and DateTime in this way.

NOTE: create koeppen_gridcode.csvt first for automated type recognition
```
db.in.ogr input=koeppen_gridcode.csv output=koeppen_gridcode
db.select table=koeppen_gridcode
```

### PR DESCRIPTION
**Pull request by neteler**
Improve CSV import example with GDAL related column type autodetection (suggested by @cmbarton).

Fixes #4674 (see also related issue #4593).

Additionally:
- minor HTML fixes in some manual pages

### CODE REVIEW

File: imagery/imageryintro.html [hunk 1]
```diff
@@ -209,11 +209,13 @@ <h3>Image classification</h3>
   <li> Supervised classification (<a href="i.gensigset.html">i.gensigset</a>,
    <a href="i.smap.html">i.smap</a>)</li>
   </ul>
+  </li>
```
Comment (echoix) [inline imagery/imageryintro.html:212]: "Radiometric classification:" doesn't close it's li inside the top ul. I double checked it with indented formatting to find what was missing,

image

So, total 3 added lines are needed, we've got 2: 
image

**Comment by echoix** [comment]
Otherwise it is good and ready to go

File: imagery/imageryintro.html (hunk 1 above)
Comment (echoix) [reply imagery/imageryintro.html:212]: There's also a typo in that file I think in "atmosphericic" ->"atmospheric"?

Comment (neteler) [reply imagery/imageryintro.html:213]: Fixed in 0e9e3849b33a538c4621021b3896df9da9604fda

Comment (neteler) [reply imagery/imageryintro.html:213]: The page now passes https://validator.w3.org/

**Review by echoix — APPROVED** [review]

**Pull request merged**